modules/op-release-notes-1-12.adoc

```diff
@@ -188,3 +188,17 @@ You must provide these parameters using secrets. After upgrading to {pipelines-s
 * Before this update, if the GitHub API provided a paginated response, {pac} used only the first page of the response. With this update, all paginated responses are processed fully.
 
 * Before this update, the {tekton-chains} controller crashed when the host address of KMS Hashicorp Vault was configured incorrectly or when {tekton-chains} was unable to connect to the KMS Hashicorp Vault. With this update, {tekton-chains} logs the connection error and does not crash.
+
+[id="release-notes-1-12-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.12.1
+
+With this update, {pipelines-title} General Availability (GA) 1.12.1 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-12-1_{context}"]
+=== Fixed issues
+
+* Before this update, when using {pac}, after you pushed changes to a pull request, the CI status check for this pull request could remain as `waiting for status to be reported` and the associated pipeline run did not complete. With this update, after you push changes to a pull request, the associated pipeline run completes normally and the CI status check for the pull request is updated.
+
+* Before this update, when using {pac}, if you created an access policy on the `Repository` custom resource (CR) that did not include a particular user and then added the user to the `OWNER` file in the Git repository, the user would have no rights for the {pac} CI process. For example, if the user created a pull request into the Git repository. the CI process would not run on this pull request automatically. With this update, a user who is not included in the access policy on the `Repository` CR but is included in the `OWNER` file, the user has full rights for the CI process for the repository.
```

Comment:
full rights is too much  a broad terms, the user will be allowed to run the CI will be sufficient..